cts-java/src/test/java/org/ga4gh/cts/api/g2p/GenotypePhenotypeSearchIT.java

```diff
@@ -0,0 +1,413 @@
+      * @throws InvalidProtocolBufferException if there's a problem processing the JSON response from the server
+      */
+     @Test
+     public void testGenotypesSearchByNameKIT() throws   InvalidProtocolBufferException, UnirestException, GAWrapperException  {
+         final String phenotypeAssociationSetId = Utils.getPhenotypeAssociationSetId(client);
+         SearchFeaturesRequest request = SearchFeaturesRequest
+                 .newBuilder()
+                 .setFeatureSetId(Utils.getFeatureG2PSetId(client))
+                 .setName(TestData.FEATURE_NAME)
+                 .build();
+
+         SequenceAnnotationServiceOuterClass.SearchFeaturesResponse response = client.sequenceAnnotations.searchFeatures(request);
+         assertThat(response.getFeaturesList()).isNotNull() ;
+         assertThat(response.getFeaturesList()).isNotEmpty() ;
+         assertThat(response.getFeaturesList().size()).isEqualTo(3) ;
+     }
+
+    /**
+     * Simple search for evidence, searches by drug name.
+     *
+     * @throws GAWrapperException if the server finds the request invalid in some way
+     * @throws UnirestException if there's a problem speaking HTTP to the server
+     * @throws InvalidProtocolBufferException if there's a problem processing the JSON response from the server
+     */
+    @Test
+    public void testGenotypePhenotypeSearchEvidence() throws InvalidProtocolBufferException, UnirestException, GAWrapperException {
+        final String phenotypeAssociationSetId = Utils.getPhenotypeAssociationSetId(client);
+        EvidenceQuery evidenceQuery = EvidenceQuery
+                .newBuilder()
+                .setDescription(TestData.EVIDENCE_NAME)
+                .build();
+        SearchGenotypePhenotypeRequest request = SearchGenotypePhenotypeRequest
+                .newBuilder()
+                .setPhenotypeAssociationSetId(phenotypeAssociationSetId)
+                .addEvidence(evidenceQuery)
+                .build();
+        SearchGenotypePhenotypeResponse response = client.genotypePhenotype.searchGenotypePhenotypes(request);
+        assertThat(response.getAssociationsList()).isNotNull() ;
+        assertThat(response.getAssociationsList()).isNotEmpty() ;
+    }
+
+    /**
+     * TODO
+     * Checks that evidence level is present, searches by drug name.
+     *
+     * @throws GAWrapperException if the server finds the request invalid in some way
+     * @throws UnirestException if there's a problem speaking HTTP to the server
+     * @throws InvalidProtocolBufferException if there's a problem processing the JSON response from the server
+     */
+    @Test
+    public void testGenotypePhenotypeSearchEnsureEvidence() throws InvalidProtocolBufferException, UnirestException, GAWrapperException {
+        // simulate user interacting with sequenceAnnotations
+        final String obfuscated = "WyJkYXRhc2V0MSIsImNnZCIsImh0dHA6Ly9vaHN1LmVkdS9jZ2QvMjdkMjE2OWMiXQ";
```

Comment:
This won't generalize. I think you need a method to get a feature ID programmatically.